docs/source/testing/mocking.md

```diff
@@ -86,65 +86,23 @@ const mocks = {
 
 The previous example uses [casual](https://github.com/boo1ean/casual), a fake data generator for JavaScript, which returns a different result every time the field is called. In other scenarios, such as testing, a collection of fake objects or a generator that always uses a consistent seed are often necessary to provide consistent data.
 
-### Using `MockList` in resolvers
+### Using lists in mocks
 
-To automate mocking a list, return an instance of `MockList`:
+To automate mocking a list, return an array of the desired length. `[...new Array(n)]` is a convenient syntax for making an array containing *n* copies of `undefined`.
 
 ```js
-const { MockList } = require('apollo-server');
+const casual = require('casual');
 
 const mocks = {
   Person: () => ({
-    // a list of length between 2 and 6 (inclusive)
-    friends: () => new MockList([2,6]),
-    // a list of three lists each with two items: [[1, 1], [2, 2], [3, 3]]
-    listOfLists: () => new MockList(3, () => new MockList(2)),
+    // a list of length between 2 and 6, using the "casual" npm module
+    // to generate a random integer
+    friends: [...new Array(casual.integer(2, 6))],
+    // a list of three lists of two items: [[1, 1], [2, 2], [3, 3]]
+    listOfLists: () => [...new Array(3)].map((i) => [...new Array(2)]),
```

Comment:
Noticed this on a drive-by
```suggestion
    listOfLists: () => [...new Array(3)].map((_, i) => [i, i]),
```

Reply:
I adapted this from the [graphql-tools docs](https://www.graphql-tools.com/docs/mocking/#using-lists-in-mocks) (which I suspect were adapted from these docs in the first place) FWIW